python/python/bystro/parent_of_origin/parent_of_origin.py

```diff
@@ -362,16 +363,59 @@ def fit(
                 else:
                     bootstrap_samples_[i] = -1 * parent_effects
 
-            if self.store_samples:
-                self.bootstrap_samples_ = bootstrap_samples_
+            alpha = 0.05
+            alpha1 = alpha / 2
+            alpha2 = 1 - alpha / 2
+            ci_eigenvector = np.zeros((self.n_phenotypes, 2))
+            for j in range(self.n_phenotypes):
+                bootstrap_vector_component = bootstrap_samples_[:, j]
+                z0_vector_component = norm.ppf(
+                    np.mean(bootstrap_vector_component < self.parent_effect_[j])
+                )
+                j_values_vector_component = np.array(
+                    [
+                        np.delete(bootstrap_vector_component, i).mean()
+                        for i in range(n_p_b)
+                    ]
+                )
+                jackknife_mean_vector_component = (
+                    j_values_vector_component.mean()
+                )
+                a_vector_component = np.sum(
```

Comment:
Could you explain the constants here (** 3, 6.0 * np.sum, **1.5, etc) or reference the equation?

Reply:
Referenced the equation, it requires a whole chapter to explain basically.